src/commands/platforms/destroy.ts

```diff
@@ -0,0 +1,74 @@
+import { flags } from '@oclif/command';
+import chalk from 'chalk';
+import { cli } from 'cli-ux';
+import inquirer from 'inquirer';
+import Command from '../../base-command';
+
+export default class PlatformDestroy extends Command {
+  static aliases = ['platform:destroy'];
+  static description = 'Destroy a platform';
+
+  static flags = {
+    ...Command.flags,
+    auto_approve: flags.boolean({
+      description: 'Automatically apply the changes without reviewing the diff',
+      char: 'a',
+      default: false,
+    }),
+  };
+
+  static args = [{
+    name: 'namespaced_platform',
+    description: 'Name of the platform to destroy',
+    required: true,
+    parse: (value: string) => value.toLowerCase(),
+  }];
+
+  async run() {
+    const { args, flags } = this.parse(PlatformDestroy);
+
+    const [account_name, platform_name] = args.namespaced_platform.split('/');
+    if (!account_name || !platform_name) {
+      throw new Error(`Please specify a namespaced platform in the form <account_name>/<platform_name>`);
+    }
+
+    let account;
+    try {
+      account = (await this.app.api.get(`/accounts/${account_name}`)).data;
+    } catch (err) {
+      throw new Error(`The account ${account_name} does not exist.`);
```

Comment:
Don't re-throw api errors. Lets expect that the api will return a human readable error msg. If it doesn't we should add an issue. Main is issue is not every error will be account doesn't exist.